### PR TITLE
CAMEL-21024 - Camel-AWS-Secrets-Manager: Eventbridge/SQS notifications documentation

### PR DESCRIPTION
**Pull request by oscerd**
# Description

# Target

- [x] I checked that the commit is targeting the correct branch (note that Camel 3 uses `camel-3.x`, whereas Camel 4 uses the `main` branch)

# Tracking
- [x] If this is a large change, bug fix, or code improvement, I checked there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).

# Apache Camel coding standards and style

- [x] I checked that each commit in the pull request has a meaningful subject line and body.

- [x] I have run `mvn clean install -DskipTests` locally and I have committed all auto-generated changes

### CODE REVIEW

File: components/camel-aws/camel-aws-secrets-manager/src/main/docs/aws-secrets-manager-component.adoc [hunk 1]
```diff
@@ -270,6 +270,66 @@ where `camel.vault.aws.refreshEnabled` will enable the automatic context reload,
 
 Note that `camel.vault.aws.secrets` is not mandatory: if not specified the task responsible for checking updates events will take into accounts or the properties with an `aws:` prefix.
 
+=== Automatic Camel context reloading on Secret Refresh with Eventbridge and AWS SQS Services
+
+There is also the possibility of using AWS Eventbridge in combination with AWS SQS Service.
```
Comment (gnodet) [inline components/camel-aws/camel-aws-secrets-manager/src/main/docs/aws-secrets-manager-component.adoc:275]: `Another option is to use AWS EventBridge in conjunction with the AWS SQS service.`

File: components/camel-aws/camel-aws-secrets-manager/src/main/docs/aws-secrets-manager-component.adoc [hunk 2]
```diff
@@ -270,6 +270,66 @@ where `camel.vault.aws.refreshEnabled` will enable the automatic context reload,
...
+
+The needed bits on the AWS side are the following:
```
Comment (gnodet) [inline components/camel-aws/camel-aws-secrets-manager/src/main/docs/aws-secrets-manager-component.adoc:277]:
```
On the AWS side, the following resources need to be created:

- an AWS Couldtrail trail
- an AWS SQS Queue
- an Eventbridge rule of this kind
```

Comment (gnodet) [reply components/camel-aws/camel-aws-secrets-manager/src/main/docs/aws-secrets-manager-component.adoc:277]: Btw, I'm not sure what `this kind` refers too, maybe it needs to be a bit more explicit ?

Comment (gnodet) [reply components/camel-aws/camel-aws-secrets-manager/src/main/docs/aws-secrets-manager-component.adoc:277]: Ah, what about `of the following kind` ?

Comment (oscerd) [reply components/camel-aws/camel-aws-secrets-manager/src/main/docs/aws-secrets-manager-component.adoc]: Done

**Review by gnodet — CHANGES_REQUESTED** [review]

**Pull request merged**